packages/components/src/components/contract-card/contract-card-items/contract-update-form.jsx

```diff
@@ -44,6 +44,11 @@ const ContractUpdateForm = props => {
         validation_errors,
     } = contract;
 
+    const [contract_profit_or_loss, setContractProfitOrLoss] = React.useState({
```

Comment:
changing input has lag because it's getting the value of inputs from contract. Adding local state for updating the view which is sync with contracts value improve this lag and fixes the performance issue.